src/App.js

```diff
@@ -0,0 +1,178 @@
+import React, { useEffect, useState } from 'react';
+import InputRange from 'react-input-range';
+import 'react-input-range/lib/css/index.css'
+import logo from './jurgisLogo.png';
+import bgimage from './dp.jpg'
+import MovieCard from './components/MovieCard.js';
+import Spinner from 'react-bootstrap/Spinner'
+import 'bootstrap/dist/css/bootstrap.min.css';
+import './App.css';
+import { Jumbotron, Button, Navbar, Nav, NavDropdown, FormControl, } from "react-bootstrap";
+import ReactModal from 'react-modal';
+import YouTube from '@u-wave/react-youtube';
+
+let apikey = process.env.REACT_APP_JURGIS;
+let keyword = '';
+let page = 1;
+let movieList = [];
+
+function App() {
+  let [movies, setMovies] = useState(null);
+  const [genres, setGenres] = useState([]);
+  let [modal, SetModal] = useState(false);
+  let [rate, setRate] = useState(0);
+  let [trailer, setTrailer] = useState('');
+
+  let popularMovies = async () => {
+    let url = `https://api.themoviedb.org/3/movie/now_playing?api_key=${apikey}&language=en-US&page=${page}`;
+    //here we get an array of 20 movies
+    let fetchedData = await fetch(url);
+    let data = await fetchedData.json();
+    movieList = movieList.concat(data.results);
+
+    movieList.map(movie => {
+      movie.genres = movie.genre_ids.map(genre => genres.find(el => el.id === genre))
+    })
+    setMovies(movieList)
```

Comment:
use a linter, you should be consistent about `;`. if you're going to use them, use them everywhere, or else don't use them anywhere.

Reply:
also this could be confusing with making sure genres is set correctly; i think you're fine though.